docs/runtimes/load-from-hub.md

```diff
@@ -1,56 +1,70 @@
+```
+
 ## Functions
 
 There are functions for ETL, data preparation, training (ML & Deep learning), serving, alerts and notifications and more.
-Each function has a docstring that explains how to use it. The functions are categorized and their associated versions are listed, so you can easily find a suitable function/module for your needs.
-
-### Prerequisiteß
-
-- {ref}`Create your project <create-projects>`
+Each function has a docstring that explains how to use it. The functions are categorized and their associated versions are listed, so you can easily find a suitable function for your needs.
 
 ### Load a function from the MLRun hub
 
-This section demonstrates how to import a function from the hub into your project, and provides some basic instructions on how to run the function.
+There are two ways to import a function from the hub:
+- {py:meth}`~mlrun.projects.MlrunProject.set_function`: adds or update a function object to your project
+- {py:meth}`~mlrun.import_function`: creates a function object that you can use as relevant, for example running it as a job
 
-Run `project.set_function` to add or update a function object to your project.
-See all the parameters in {py:meth}`~mlrun.projects.MlrunProject.set_function`.
+#### Use `set_function`
 
-The `describe` function analyzes a csv or parquet file for data analysis.
-To load the `describe` function from the MLRun hub:
+This example runs the describe function. This function analyzes a dataset (in this case it's a csv file) and generates HTML files (e.g. correlation, histogram) and saves them under the artifact path.
 
 ```python
+# Load the `describe` function from the MLRun hub:
 project.set_function("hub://describe", "describe")
-```
 
-To load the same function from your [custom hub](#custom-hub):
-```python
+# Or load the same function from your [custom hub](#custom-hub):
+
 project.set_function("hub://<hub-name>/describe", "describe")
-```
-```{caution} 
-If you don't specify a hub name at all, the algorithm searches for the function in all the hubs, 
-giving preference to newly defined hubs. Therefore, if you 
-have multiple hubs, best practice is to explicitly mention the hub name.
+
+# Create a function object named, for example, `my_describe`:
+my_describe = project.func("describe")
 ```
 
-After loading the function, create a function object named, for example, `my_describe`:
+#### Use import_function
+This example uses the aggregate function, which perform a rolling aggregation of artifacts.
 
-```python
-my_describe = project.func("describe")
 ```
+# Import the function
+aggregate_function = mlrun.import_function("hub://aggregate")
+if os.getenv('V3IO_ACCESS_KEY','FALSE')=='TRUE':
```

Comment:
Why do we need all of that code? we are talking about importing the aggregate function and then we add more lines without any context? if these lines are required for demonstrate a running example later, then it should be clear. We should move these lines to the running example or at least explain we do we need them.